pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -50,9 +53,9 @@ const (
 	burstingThroughputMode = awsefs.ThroughputModeBursting
 	defaultThroughputMode  = burstingThroughputMode
 
-	efsType            = "EFS"
-	k10BackupVaultName = "k10vault"
-	testMarker         = ""
+	efsType                   = "EFS"
+	defaultK10BackupVaultName = "k10vault"
+	testMarker                = ""
```

Comment:
What's this? Should be removed?

Reply:
This should also be there. This is the default value in case it's set.